Tests/XcodeTests/XcodebuildBuildProjectTest.swift

```diff
@@ -1,11 +1,14 @@
 import Foundation
-import XCTest
 import Shared
 @testable import XcodeSupport
+import XCTest
 
-class XcodebuildBuildProjectTest: XCTestCase {
-    var xcodebuild: Xcodebuild!
-    var project: XcodeProject!
+// swiftlint:disable single_test_class
+final class XcodebuildBuildProjectTest: XCTestCase {
+    // swiftlint:disable implicitly_unwrapped_optional
```

Comment:
Let's disable `implicitly_unwrapped_optional` for all tests. This is a common pattern to prevent memory leaks in xctests.

Reply:
So I think we can do that with a `swiftLint.yml` at the top of the Tests directory - I should probably do that on my own codebases - I have a *lot* of `// swiftlint:disable implicitly_unwrapped_optional`

Reply:
Yeah that should work, we can do that in a follow-up PR.